draft-ietf-quic-recovery.md

```diff
@@ -244,11 +249,9 @@ ack-eliciting packet. QUIC recovery algorithms do not assume the peer sends
 an ACK immediately when receiving a second ack-eliciting packet.
 
 In order to accelerate loss recovery and reduce timeouts, the receiver SHOULD
-send an immediate ACK when it receives a new packet which is not one greater
-than the largest received packet number. A receiver MAY send immediate ACKs
-for the next few ack-eliciting packets that are received, but SHOULD NOT
-send an immediate ACK for more than 1/8 RTT after receiving an out-of-order
-packet.
+send immediate ACKs for an interval after it receives an out-of-order packet.
```

Comment:
The SHOULD is intended to apply to just the first immediate ACK, as with ECN marks.

Reply:
So I'm still not clear.  If I get a continuous stream of reordered packets, a) when do I cease immediate ACKs, and b) if I do stop immediate ACKs, when can I start again?

Reply:
If every packet is reordered(ie: every other packet is missing), then you keep sending immediate ACKs.

Reply:
So it isn't the first immediate ACK, it's every out-of-order packet that starts (or refreshes) a 1/8 RTT timer.  That helps.